### PR TITLE
add normalization page

### PR DESCRIPTION
**Pull request by alozowski**

### CODE REVIEW

File: docs/source/en/open_llm_leaderboard/normalization.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+# Scores Normalization
+
+This page explains how scores are normalized in our leaderboard, particularly for tasks with subtasks like MUSR and BBH.
+
+## General Normalization Process
+
+For tasks without subtasks (e.g., GPQA, MMLU-PRO), normalization is straightforward:
+1. Subtract the random score
+2. Remap to the range (0, 1)
```
Comment (clefourrier) [inline docs/source/en/open_llm_leaderboard/normalization.md:9]: Give the formula from the code

File: docs/source/en/open_llm_leaderboard/normalization.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+## Normalizing Tasks with Subtasks
+
+To normalize results for tasks with subtasks, follow these steps:
+
+1. Define a normalization function:
+
+```
+def normalize_within_range(value, lower_bound, higher_bound):
+    return (value - lower_bound) / (higher_bound - lower_bound)
+```
```
Comment (clefourrier) [inline docs/source/en/open_llm_leaderboard/normalization.md:22]: This formula could go above

File: docs/source/en/open_llm_leaderboard/normalization.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+
+## Normalizing Tasks without Subtasks
+For tasks without subtasks (e.g., GPQA, MMLU-PRO), the normalization process is straightforward:
+- Determine the lower bound (random guess baseline).
+- Apply the normalization function.
+- Scale to a percentage.
+
+### Example: Normalizing GPQA Scores
+GPQA has 4 `num_choices`, so the lower bound is 0.25.
```
Comment (clefourrier) [inline docs/source/en/open_llm_leaderboard/normalization.md:33]:
```suggestion
GPQA has 4 `num_choices`, so the lower bound is 0.25 (1/`num_choices` = 1/4 = 0.25).
```

File: docs/source/en/open_llm_leaderboard/normalization.md [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+```
+
+## Generative Evaluations
+Generative evaluations like MATH and IFEval require a different approach:
+1. **MATH:** Uses exact match accuracy. The lower bound is effectively 0, as random guessing is unlikely to produce a correct answer.
+2. IFEval:
+    - For instance-level evaluation (`ifeval_inst`), we use strict accuracy.
+    - For prompt-level evaluation (`ifeval_prompt`), we also use strict accuracy.
+    - The lower bound for both is 0, as random generation is unlikely to produce correct answers.
+3. BBH: Uses normalized accuracy (`acc_norm`). The lower bound varies by subtask, based on the number of choices.
```
Comment (clefourrier) [inline docs/source/en/open_llm_leaderboard/normalization.md:102]: Not a generative evaluation

Comment (alozowski) [reply docs/source/en/open_llm_leaderboard/normalization.md]: yep, my bad, corrected!

**Pull request merged**